storage/rocksdb.go

```diff
@@ -224,8 +225,15 @@ func (store *rocksStore) init() error {
 
 func (store *rocksStore) GetQueue(name string) (Queue, error) {
 	if name == "" {
-		return nil, fmt.Errorf("Queue name cannot be blank")
+		return nil, fmt.Errorf("queue name cannot be blank")
 	}
+	
+	valid := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_.-"
+	invalid := strings.Trim(name, valid)  // strip the valid chars, leaving just the bad ones
```

Comment:
Can you make this a regexp, like LAST_ELEMENT in webui/pages.go?

Reply:
For performance reasons, would it be better to add the check after the fast path return here:

```go
	if ok {
		return q, nil
	}
```

Reply:
Makes sense, I've updated the PR